nf_core/lint.py

```diff
@@ -103,21 +104,59 @@ def pf(file_path):
                 self.warned.append((1, "File not found: {}".format(files)))
 
 
+    def check_docker(self):
+        """minimal tests only"""
+        logging.debug('Checking Dockerfile')
+        fn = os.path.join(self.path, "Dockerfile")
```

Comment:
We should check if this exists first. Will probably get a nasty exception here and halt all testing if not.

More generally - should think about making the test for Docker file(s) more flexible given that some pipelines may not be able to have one single file.

Reply:
Okay. Since existance is checked elsewhere I return silently in the newest commit